nixos/modules/services/networking/fedimintd.nix

```diff
@@ -268,9 +331,12 @@ in
               RestrictAddressFamilies = [
                 "AF_INET"
                 "AF_INET6"
+                "AF_UNIX"
+                "AF_NETLINK"
```

Comment:
Only `AF_NETLINK` was strictly required for iroh iirc. https://github.com/n0-computer/iroh/issues/3265